kuksa-client/kuksa_client/grpc/__init__.py

```diff
@@ -635,9 +661,21 @@ def _raise_if_invalid(self, response):
                 err, preserving_proto_field_name=True) for err in response.errors]
         else:
             errors = []
-        if (error and error['code'] is not http.HTTPStatus.OK) or any(
-            sub_error['error']['code'] is not http.HTTPStatus.OK for sub_error in errors
-        ):
+
+        raise_error = False
+        if (error and error.get('code') != 200):
```

Comment:
I noticed in tests that we never will be equal to http.HTTPStatus.OK as it gets translated to the corresponding number